src/elements/digit-custom-dialog/DigitCustomDialog.element.jsx

```diff
@@ -37,8 +38,12 @@ const DigitCustomDialog = ({
             aria-labelledby="alert-dialog-title"
             aria-describedby="alert-dialog-description"
         >
-            <DialogTitle id="alert-dialog-title">{title}</DialogTitle>
-            <DialogContent>{renderMain()}</DialogContent>
+            <DialogTitle id="alert-dialog-title">
+                {title}
+                {renderTitle != null && renderTitle(confirm, cancel)}
+            </DialogTitle>
```

Comment:
I think it would be preferable to wrap this inside a new `<DialogTitleContainer />` since I'm pretty sure `<DigitTitle />` mostly handles font things. This might cause fonts in `renderTitle` to be changed unexpectedly. 

Also being able to access the css via prop, in the same manner as `detailsProps` in `<DigitCrud />` would be nice. This could also be added for `<DialogContent />` and `<DialogActions />`.